net/netxng/src/RRawFileNetXNG.cxx

```diff
@@ -0,0 +1,107 @@
+// @(#)root/io:$Id$
+// Author: Michal Simon
+
+/*************************************************************************
+ * Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include "ROOT/RRawFileNetXNG.hxx"
+
+#include <TError.h>
+
+#include <memory>
+#include <stdexcept>
+#include <vector>
+#include <XrdCl/XrdClFile.hh>
+#include <XrdCl/XrdClFileSystem.hh>
+
+
+namespace {
+constexpr int kDefaultBlockSize = 128 * 1024; // Read in relatively large 128k blocks for better network utilization
+} // anonymous namespace
+
+namespace ROOT {
+namespace Internal {
+
+struct RRawFileNetXNGImpl {
+   RRawFileNetXNGImpl() = default;
+   RRawFileNetXNGImpl(const RRawFileNetXNGImpl &) = delete;
+   RRawFileNetXNGImpl &operator=(const RRawFileNetXNGImpl &) = delete;
+   ~RRawFileNetXNGImpl() = default;
+
+   XrdCl::File file;
+};
+
+} // namespace Internal
+} // namespace ROOT
+
+
+ROOT::Internal::RRawFileNetXNG::RRawFileNetXNG( std::string_view   url,
+                                                RRawFile::ROptions options )
+   : RRawFile( url, options ), pImpl( new RRawFileNetXNGImpl() )
+{
+}
+
+ROOT::Internal::RRawFileNetXNG::~RRawFileNetXNG()
+{
+}
+
+std::unique_ptr<ROOT::Internal::RRawFile> ROOT::Internal::RRawFileNetXNG::Clone() const
+{
+   return std::make_unique<RRawFileNetXNG>( fUrl, fOptions );
```

Comment:
Add `utility` include

Reply:
AFAIK the `make_unique` utility comes from `#include <memory>`